### ISSUE TITLE
Primera propuesta de API - ¿cómo ven?

### ISSUE THREAD

**Issue by rodowi** [mxabierto/validadores#1]
La propuesta del API para pedir validaciones de datasets a "La Validadora" (un motor de validaciones).

https://github.com/mxabierto/validadores/wiki/API

¿Qué opinan?

**Comment by juanpabloe** [comment]
ayudaría mucho poder ver example requests & responses en cada método

**Comment by rodowi** [comment]
:+1: 

**GET** /ping
```json
{
  "response": "pong"
}
```

**Comment by miguelsalazar** [comment]
Sí, está totalmente contemplado para la definición completa del API. Iré agregando lo que ya tengamos definido.
